00-Raster-Structure.Rmd

```diff
@@ -353,44 +346,107 @@ GDALinfo("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
 
 ```
 
-## Bad data values
-
-Bad data values are different from NoData values. Sometimes a raster's metadata
-will tell us a value range of values for the raster. Sometimes, we need to use
-some practial scientific insight as we examine the data - just as we would for field
-data. 
+2. **Bad data values:** Bad data values are different from NoData values. 
```

Comment:
let's do it in the min/max - we can just bring it up. If we use an elevation surface we can say - hey - know any spots in the US that are 40,000m tall?? yea - no. i can just inject a few bad pixels in there!!  Then we can talk about it and suggest -- hmmm why don't we set everything >40000 to na? as one idea...